corehq/apps/locations/static/locations/js/widgets.js

```diff
@@ -145,6 +147,50 @@ hqDefine("locations/js/widgets", [
         $select.trigger('select-ready');
     }
 
+    function updateScreenReaderNotification(notificationText, notificationElementId) {
+        const srNotification = $(notificationElementId);
+        if (notificationText) {
+            srNotification.html("<p>" + notificationText + "</p>");
+            srNotification.attr("aria-live","assertive");
+        } else {
+            srNotification.attr("aria-live","off");
+        }
+    }
+
+    function handleLocationWarnings(select) {
+        const requestDomain = initialPageData.get('domain');
+        const selectedLocations = select.val();
+        const requestUserDomainMemberships = initialPageData.get('request_user_domain_memberships');
+        const requestUserLocations = _.find(requestUserDomainMemberships, function (dm) {
+            if (dm.domain === requestDomain) {
+                return dm.assigned_location_ids;
+            }
+        });
+        let shareLocations = false;
+        _.every(selectedLocations, function (locationId) {
+            if (_.contains(requestUserLocations, locationId)) {
+                shareLocations = true;
+            }
+        });
+
+        const noCommonLocationWarning = $(".no-common-location");
```

Comment:
I think an id would be more appropriate than a class to identify a single element.

Reply:
agreed, changed here https://github.com/dimagi/commcare-hq/pull/35379/commits/fce39404d26a4dd6572e5ff93b961ec797c409f3